README.md

```diff
@@ -5,11 +5,18 @@ This is a project template providing a great out-of-the-box experience for new D
 
 This project includes the following packages and configuration:
 * Drupal core
-* [Acquia CMS](https://github.com/acquia/acquia_cms) (Drupal distribution)
+* Drupal core scaffold
+* [Acquia CMS](https://github.com/acquia/acquia-cms-starterkit) (Starter kit)
 * [Drush](https://github.com/drush-ops/drush) (Drupal CLI and development tool)
-* [Acquia Connector](https://drupal.org/project/acquia_connector) (Management and reporting tool for Acquia Cloud sites)
-* [Acquia Site Studio](https://www.acquia.com/products/drupal-cloud/site-studio) (Low-code visual page builder)
 * [Asset Packagist](https://asset-packagist.org/) repository, package, and configuration
+* Acquia environment detection
+* Acquia platform memcache settings
+* Best practices for Drupal development, testing and project architcture
+
+The Acquia CMS starter kit  allows you install Drupal for a given style of CMS:
+* Acquia Enterprise Low Code for low code CMS with Acquia Site Studio.
+* Acquia CMS Headless for CMS configurations in micro services environments.
+* Acquia CMS Community for more minimal starting points allowing greater customization.
```

Comment:
This copy seems awkward. It would be better if I could scan the list for the names of each.
```suggestion
* **Low Code** for low code CMS with Acquia Site Studio.
* **Headless** for CMS configurations in micro services environments.
* **Community** for more minimal starting points allowing greater customization.
```
It's also a confusing list to me. Both 1 & 3 seem to be "minimal". What do I choose for a "normal"/"standard" install? Also, what about the third option is "community" focused? Shouldn't it just be called "Minimal"?

Reply:
Currently Community/Minimal is Normal.  This has been named community after a lot of internal discussions. So we have low code which is enterprise, Community and Headless.

Reply:
Updated the README with better description.